app/views/compute_resources_vms/form/openstack/scheduler_filters/_raw.erb

```diff
@@ -0,0 +1,2 @@
+<%=
+  textarea_f f, :scheduler_hint_value, :help_block => _("JSON object of the scheduler hint"), :size => "col-md-8", :rows => "6", :class => "no-stretch", :label => _("Server hint data"), :disabled => !new_host %>
```

Comment:
@ohadlevy The idea was to rely as less as possible on the RB code and have these filters more "plugable", thought it would be easier to "read" if all there is to do, is just select the proper scheduler filter "piece" and put it in place, as it is.